app/src/main/java/com/lagradost/cloudstream3/ui/result/ResultViewModel2.kt

```diff
@@ -113,6 +132,18 @@ data class ResultData(
     val plotHeaderText: UiText,
 )
 
+data class CheckDuplicateData(
+    val name: String,
+    val year: Int?,
+    val imdbId: String?
```

Comment:
make this include the entire syncdata mapping

Reply:
Thanks! Done.